google_compute_engine/metadata_scripts/script_retriever.py

```diff
@@ -50,6 +50,18 @@ def _DownloadGsUrl(self, url, dest_dir):
     Returns:
       string, the path to the file storing the metadata script.
     """
+    try:
+       subprocess.check_call(
+         ['which', 'gsutil'],
+         stdout=subprocess.PIPE,
+         stderr=subprocess.PIPE
+       )
+    except subprocess.CalledProcessError:
+      self.logger.warning(
+        'gsutil is not installed, cannot download items from Google Storage'
+      )
```

Comment:
OK, new test added, existing tests modified. Should pass now, lets see what happens.

Have to have legal involved for the contributor agreement signature, thus that part might take a while.